settings/settings.php

```diff
@@ -62,6 +62,7 @@
   'manatime_feature_site_template',
   'manatime_feature_tasks',
   'manatime_feature_time_entries',
+  'manatime_feature_timesheet',
```

Comment:
For consistency, I'd make the feature name plural. Ping me when you're going to do this to help you.